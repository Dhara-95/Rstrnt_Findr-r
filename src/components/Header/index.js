import React from "react";

import { LabelsObject } from "../../Labels";

const Header = () => {
  const { headerLabel } = LabelsObject;

  return <h1 className="text-5xl text-grey py-6">{headerLabel}</h1>;
};

export default Header;
