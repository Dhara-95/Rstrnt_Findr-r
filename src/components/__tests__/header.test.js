import React from "react";
import renderer from "react-test-renderer";

import { Header } from "../Header";

it("displays the Header correctly", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
