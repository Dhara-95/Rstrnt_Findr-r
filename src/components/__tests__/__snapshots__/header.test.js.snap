// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the Header correctly 1`] = `
<h1
  className="text-5xl text-grey py-6"
>
  Rstrnt Findr-r
</h1>
`;
